common-components/schemas/type/money.md

```diff
@@ -0,0 +1,36 @@
+# Money
+
+The `Money` type represents an amount of money in a specified currency.
+
+## Schema
+
+A `Money` has two fields:
+
+- The `currency_code` field is a string containing a currency code.
+
+  The three-letter currency codes defined in the ISO 4217 standard are always
+  supported.
+
+  APIs may define additional currency codes that are not included in ISO 4217
+  (for example, virtual currencies or cryptocurrencies). These must start with
+  a `.` character, in order to distinguish them from potential future ISO 4217
+  codes.
+
+  Examples:
+
+  - "USD" - ISO 4217 code for United States dollar
+  - ".BTC" - Potential API-defined extension for Bitcoin.
+  - ".RBX" - Potential API-defined extension for the virtual currency Robux
+
+- The `amount` field is a field of type [`Decimal`][], representing the amount
+  of currency.
+
+## Examples
+
+- Five US dollars === `{currency_code: "USD", {digits: 5}}`
```

Comment:
should these be `amount` now?

Reply:
They should have been before but I messed up.  Fixed (and more generally updated all the examples for consistency).

Reply:
They should have been `amount` before too - I made a mistake in the original draft.  Fixed.